lib/grammers-mtsender/src/lib.rs

```diff
@@ -276,13 +251,9 @@ impl<T: Transport, M: Mtp> Sender<T, M> {
         {
             Ok(_) => {
                 self.read_buffer.clear();
-                self.read_index = 0;
                 self.process_mtp_buffer().map_err(|e| e.into())
             }
             Err(transport::Error::MissingBytes(n)) => {
```

Comment:
```suggestion
            Err(transport::Error::MissingBytes(_)) => {
```

I wonder if it's worth keeping the amount `n` at all. If we don't ignore it, perhaps we could call `.reserve()` instead? But if the buffer already has enough size it's pointless. Anyway as is this is a warning I'm pretty sure.

Reply:
I was thinking of entirely removing the calculation of it; I think `.reserve()` is already called in the most optimal way regardless